pkg/controllers/sidecar/config/config.go

```diff
@@ -203,10 +222,29 @@ func (s *ScyllaConfig) setupEntrypoint(ctx context.Context) (*exec.Cmd, error) {
 		if err := s.validateCpuSet(ctx, cpusAllowed, shards); err != nil {
 			return nil, errors.WithStack(err)
 		}
-		args = append(args, fmt.Sprintf("--cpuset=%s", cpusAllowed))
+		args["cpuset"] = &cpusAllowed
+	}
+
+	if len(cluster.Spec.ScyllaArgs) > 0 {
+		for argName, argValue := range convertScyllaArguments(cluster.Spec.ScyllaArgs) {
```

Comment:
It doesn't work for older Scylla versions, it doesn't even work for released versions. We definitely need some kind of warning or info log level message about missing support.

Reply:
Validation has been added.